ansible/roles/operator-pipeline/tasks/webhook-event-listener.yml

```diff
@@ -0,0 +1,127 @@
+---
```

Comment:
I think this and other files from this PR should not be stored in `tasks` directory because it is not a tekton task. Can you make separate dir for triggers?

Reply:
Aha, i didn't noticed it is a separate task directory.